config/rbac/dnses_clusterrole_binding.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: dnses-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: dnses-role
+subjects:
+  - kind: ServiceAccount
+    name: default
```

Comment:
Avoid to use 'default' service account for the controller workload. Associate to a specific service account created into the rbac.